receiver/faroreceiver/keyval.go

```diff
@@ -0,0 +1,87 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package faroreceiver
+
+import (
+	"fmt"
+	"sort"
+
+	om "github.com/wk8/go-ordered-map"
+)
+
+// KeyVal is an ordered map of string to interface
+type KeyVal = om.OrderedMap
+
+// NewKeyVal creates new empty KeyVal
+func NewKeyVal() *KeyVal {
```

Comment:
We should probably unexport specific to this component struct and functions or move it to the internal package.